src/Cli.elm

```diff
@@ -197,39 +197,71 @@ generateFileFromOpenApiSpec config apiSpec =
             )
         |> BackendTask.andThen
             (\outputPath ->
+                let
+                    pad4 : String -> String
+                    pad4 =
+                        String.padLeft 4 ' '
```

Comment:
Personally I prefer `String.padLeft` since it's more descriptive and I cannot tell from the `"...."` version how many spaces it represents except for the function name and if I accidentally add a space or remove one in the future then the name meaning is broken also. You know what I mean?

Reply:
I created a middleground proposal instead.

Reply:
I'd be fine with something like
```elm
indent4 : String -> String
```

Reply:
True, my bad, updating now. 👍🏻

Reply:
Updated 😄!